src/main/scala/diplomaticobjectmodel/model/OMRTLModule.scala

```diff
@@ -2,6 +2,47 @@
 
 package freechips.rocketchip.diplomaticobjectmodel.model
 
+// Examples of ClockRelationships
+//  {
+//   "clock0": "core_clock_0",
+//    "clock1": "clock",
+//    "relationship": "$clock0 == m * $clock1 where m is a positive integer",
+//  }
+
+//  {
+//    "clock1": "rtc_toggle",
+//    "relationship": "$clock0 >= 2 * $clock1",
+//  }
+
+trait OMStatus extends OMSignal {
+  name: String
+  description: Option[String]
+}
+
+trait OMCoreStatus extends OMStatus {
+  name: String
+  description: Option[String]
+  def hartId: Int
+}
+
+case class Halt(
+  name: String,
+  description: Option[String],
+  hartId: Int
+) extends OMCoreStatus
```

Comment:
Don't you need to add in the `_types` property with the default value or else the serialized object model JSON file will lack the type information associated with each of these classes?